projects/packages/videopress/src/client/admin/components/video-card/index.tsx

```diff
@@ -121,7 +121,15 @@ export const VideoThumbnail: React.FC< VideoThumbnailProps & VideoThumbnailDropd
 				</div>
 			) }
 
-			<img src={ thumbnail } alt={ __( 'Video thumbnail', 'jetpack-videopress-pkg' ) } />
+			{ thumbnail && (
+				<img src={ thumbnail } alt={ __( 'Video thumbnail', 'jetpack-videopress-pkg' ) } />
+			) }
```

Comment:
Any reason to not use ternary here?

```
thumbnail ? ... : ...
```

Reply:
good call. I changed the conditions again and again and didn't realize about the ternary, which is a better way. Updating...

Reply:
https://github.com/Automattic/jetpack/pull/26124/commits/e9c2f36bf37f49a1d65a8af15e1953d51cef9fcd thanks